### DISCUSSION TITLE
Any plans for HTML to PDF?

### DISCUSSION THREAD

**Discussion by ahylton-maharam** [QuestPDF/QuestPDF#59] [Ideas]
Are there any current plans to implement an html to pdf feature? 
I've been going through the guide and while there is an already impressive collection of features, that one is something I always look for/ask about.

**Comment by MarcinZiabek** [comment]
I am not planning to support HTML to PDF conversion. The whole point of this library is to avoid all the mess associated with this process. If you still want to use HTML, there are plenty of solutions already available 😀

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
